release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -83,6 +83,29 @@ You can now remove or rotate the GCP admin-level credential that the xref:../ope
 [id="ocp-4-7-storage"]
 === Storage
 
+[id="ocp-4-5-persistent-storage-csi-gcp-pd"]
+==== Persistent storage using the GCP PD CSI Driver Operator (Technology Preview)
+
+The Google Cloud Platform (GCP) persistent disk (PD) CSI driver is automatically deployed and managed on GCP environments, allowing you to dynamically provision these volumes without having to install the driver manually. The GCP PD CSI Driver Operator that manages this driver is in Technology Preview.
+
+[id="ocp-4-5-persistent-storage-csi-cinder"]
+==== Persistent storage using the OpenStack Cinder CSI Driver Operator (Technology Preview)
+
+You can now use CSI to provision a persistent volume using the CSI driver for OpenStack Cinder.
+
+// Add [OpenStack Cinder CSI Driver Operator] link if docs are added to Storage --> CSI for OCP 4.7:
+
+[id="ocp-4-7-storage-csi-snapshots"]
+==== Persistent storage using CSI volume snapshots is generally availabile
+
+You can use the Container Storage Interface (CSI) to create, restore, and delete a volume snapshot when using CSI drivers that provide support for volume snapshots. This feature was previously introduced as a Technology Preview feature in {product-title} 4.4 and is now generally available and enabled by default in {product-title} 4.7.
```

Comment:
In regards to how these topic sections are nested, it might not be a bad idea to put this section first. Mainly because you spell out the CSI acronym in this doc. In the two other sections CSI is never spelled out, so the user won't know what the acronym is unless they have prior knowledge or read down.  In it's current state, spelling out the CSI acronym in the GCP topic (since it appears first) would be wordy with all the other acronyms present in the first sentence.

I'm not sure if there was a specific reason for nesting in this order, but if it's not set in stone I would go in the following order for acronym-sake:

- GA- CSI
- TP-GCP
- TP-OpenStack
- vSphere

Reply:
I agree! I generally try ordering them by impact and by similarities, although the logic is fuzzy. :) You're right though that GCP PD CSI all spelled out is not great, so I've rearranged as you suggest. Thanks.